external-resolver/src/main/scala/org/genivi/sota/resolver/db/ForeignPackages.scala

```diff
@@ -44,7 +44,7 @@ object ForeignPackages {
     def * = (device, name, version, lastModified) <> (fromTuple, toTuple)
   }
 
-  private val foreignPackages = TableQuery[InstalledForeignPackageTable]
+  val foreignPackages = TableQuery[InstalledForeignPackageTable]
```

Comment:
I was planning to remove it afterwards, it saves having to create a new temporary method. I don't mind adding a method if you prefer though.